scripts/README.md

```diff
@@ -0,0 +1,324 @@
+# Local Scripts
```

Comment:
We should have a list of things needed in order to run locally. I think it's
```
apt install clang-format-18 
apt install markdownlint
apt install shellcheck
apt install yamllint
apt install cmake-format
```
?

Reply:
Pushed that into `scripts/README.md`.